### PR TITLE
Start adding Material design Components

### PR DESCRIPTION
**Pull request by saranshdhingra**
This PR adds a top bar as per the new design which uses Material Design Components in favour of the currently used Material Design Lite.

Check this for difference: https://material.io/develop/web/guides/migrating-from-mdl

Screenshot:
gcping-mdc-migration

### CODE REVIEW

File: web/src/index.html [hunk 1]
```diff
@@ -17,32 +17,42 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
     />
-    <link rel="stylesheet" href="css/gcping.css" />
-    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
```
Comment (saranshdhingra) [inline web/src/index.html:21]: Along with this, we should be able to remove the CSS from L:12 once we are done with the material design changes.

**Comment by enocom** [comment]
@saranshdhingra Mind adding some screenshots to this PR?

**Comment by crwilcox** [comment]
Can you adjust the colors for the table header and start/stop? I don't think they are in the same palette making this look a bit unfinished.

**Review by enocom — CHANGES_REQUESTED** [review]
A few visual tweaks:

1. The links in the upper right need some left padding between the text and the icons.
2. GCPing has a favicon. Could we use that as a logo next to the "GCPing" title? It looks a little bare otherwise.
3. The existing site has "Measure your latency to Google Cloud regions" which is dropped here. Can we bring that back?
4. Also, the stop button's teal clashes with the blue. Could we switch the color of the bottom to match the header?
5.

**Comment by saranshdhingra** [comment]
Thanks @crwilcox @enocom for the color and other gotchas.
Here is the updated design with the logo:
gcping w/ logo

I wanted to try a "white version" of the icon, haven't yet done so. If you think this looks good for now, I'll proceed.

The heading, "Measure your latency to Google Cloud regions" is not like in the design as a lot of elements are not yet there. It should match our internal design in the following PRs.

**Review by enocom — APPROVED** [review]
Looks good to me.

I agree with @crwilcox that the table header color classes with the blue bar. But perhaps we can fix that in the next PR and then deploy when this is all done.

**Comment by saranshdhingra** [comment]
> Looks good to me.
> 
> I agree with @crwilcox that the table header color classes with the blue bar. But perhaps we can fix that in the next PR and then deploy when this is all done.

I agree, I did change the button, somehow completely missed the table header.
I'll make sure the next PR fixes that as it will mostly be changes to the table component.

**Comment by crwilcox** [comment]
Agree, not a huge fan of that grey/blue/green header, but the worst bit is resolved. I think we may want to have a palette discussion and go from there. Thanks for getting this in @saranshdhingra

**Pull request merged**